### PR TITLE
feat(slidesInView): separation of concerns + added method to public api

### PR DESCRIPTION
**Pull request by pimdewit**
I thought `slidesInView` would be a nice addition to the public API, it is also a nice opportunity for separation of concerns. I extracted the snippet from `_updateSlidesA11y` and added it to its own getter.

### CODE REVIEW

File: test/selected-slides-per-view.js [hunk 1]
```diff
@@ -38,5 +38,13 @@
...
       expect(this.slider.selected).to.equal(numberOfSlides - 1);
     });
+
+    it('returns indexes of slides in view', async function() {
+      this.slider.selected = 0;
+      this.slider.slidesPerView = 3;
+
+      await wcutils.flush();
+      expect(this.slider.slidesInView).to.deep.equal([0, 1, 2]);
+    });
```
Comment (ciampo) [inline test/selected-slides-per-view.js]: We need more tests:
- with different values of selected (not just `0`)
- with different values of `slidesPerView` (definitely at least `1` as well)

And then we need to test for edge cases:
- what if `selected` is set on purpose out of range? (e.g. there are 5 slides, and we set `selected` to `7`)
- what if `slidesPerView` is set to `0`?
- what if looping is enabled, and the `selected` slide is the last? `slidesInView` should be `[last, first...]`


Any anything else that comes to mind
